src/popup/pages/AccountDetails.vue

```diff
@@ -167,16 +177,55 @@ export default defineComponent({
       }
     });
 
+    const showSearchBar = computed<boolean>(() => !!(
+      searchTerm.value || (
+        clientHeight.value > initialClientHeight.value
+        && appInnerScrollTop.value >= 120
+      )
+    ));
+
+    const resizeObserver = new ResizeObserver(debounce((entries) => {
+      if (!Array.isArray(entries) || !entries.length) {
+        return;
+      }
+      const _clientHeight = entries[0].target.clientHeight;
+      if (_clientHeight && (
+        (clientHeight.value + 400) < _clientHeight || _clientHeight <= initialClientHeight.value)
+      ) {
+        clientHeight.value = _clientHeight;
+      }
+    }, 100));
+    const routerBeforeEach = root.$router.beforeEach((to, from, next) => {
+      if (to.name && to.name.includes('account-details')) {
+        clientHeight.value = 0;
+      }
+      next();
+    });
+
     onMounted(() => {
       if (IS_CORDOVA) {
         window.StatusBar.backgroundColorByHexString('#191919');
       }
+      if (accountDetails.value && accountDetails.value.parentElement) {
+        resizeObserver.observe(accountDetails.value);
+        initialClientHeight.value = accountDetails.value.parentElement.clientHeight;
+        accountDetails.value.parentElement.addEventListener('scroll', () => {
+          appInnerScrollTop.value = accountDetails?.value?.parentElement?.scrollTop ?? 0;
+          clientHeight.value = accountDetails?.value?.clientHeight ?? 0;
```

Comment:
I'd suggest to extract `accountDetails.value.parentElement` to a constant to not repeat it every time